### DISCUSSION TITLE
Is there a large axum example?

### DISCUSSION THREAD

**Discussion by conways-glider** [tokio-rs/axum#1513] [Q&A] [answered]
Hello! I was wondering if there was an example of a larger axum project?

I'm really curious as to what a production server with several routes and error handling would look like.

**Comment by jamesrampton** [comment, marked as answer]
There are plenty mentioned in https://github.com/tokio-rs/axum/blob/main/ECOSYSTEM.md

The one I'm personally following for inspiration is: https://github.com/launchbadge/realworld-axum-sqlx which is commented extremely well. There's a few API changes with that repo if you're using 0.6.0-rc.2 though, so also see https://github.com/davidpdrsn/realworld-axum-sqlx

**Reply by conways-glider** [reply]
Thank you!! I can't believe I missed that file!
